jcl/src/java.base/share/classes/java/lang/invoke/MethodTypeHelper.java

```diff
@@ -177,7 +177,13 @@ static String getBytecodeStringName(Class<?> c) {
 		if (c.isArray()) {
 			return name;
 		}
```

Comment:
avoid multiple return statements and use `else if`.
```suggestion
		if (c.isArray()) {
		} else if (c.isPrimitiveClass()) {
```